web/owlspa/src/actions/serverSavePatch.js

```diff
@@ -63,16 +65,16 @@ const serverSavePatch = (patch, options = {}) => {
             } else {
               throw new Error('Error saving patch');
             }
-          } 
+          }
           return json;
         });
       })
       .then( json => {
-        
+
         if(json._id){
           dispatch({type: PATCH_SAVED});
           if(method === 'POST'){
-            dispatch(cleanUpTmpPatchFiles(json._id)).then(() => {            
+            dispatch(cleanUpTmpPatchFiles(json._id)).then(() => {
```

Comment:
it would actually be great if we didn't have to send a request from the front end to tell the back end to clean up the temp patch files after successful patch creation, but it's probably worth raising in a separate issue to resolve.